src/Features/CSharp/Portable/Completion/CompletionProviders/ExplicitInterfaceTypeCompletionProvider.cs

```diff
@@ -44,14 +44,6 @@ internal override bool IsInsertionTrigger(SourceText text, int insertedCharacter
         protected override (string displayText, string suffix, string insertionText) GetDisplayAndSuffixAndInsertionText(ISymbol symbol, SyntaxContext context)
             => CompletionUtilities.GetDisplayAndSuffixAndInsertionText(symbol, context);
 
-        protected override async Task<SyntaxContext> CreateContextAsync(
-            Document document, int position, CancellationToken cancellationToken)
-        {
-            var semanticModel = await document.ReuseExistingSpeculativeModelAsync(position, cancellationToken).ConfigureAwait(false);
```

Comment:
the reason to make this change was that all subclasses needed to know to call this special mechanism to get a speculative model.  that logic is moved up into the base class and cannot be overridden.